packages/rest-api/src/tests/getBridgeLimits.test.ts

```diff
@@ -0,0 +1,81 @@
+import request from 'supertest'
+import express from 'express'
+
+import getBridgeLimitsRoute from '../routes/bridgeLimitsRoute'
+
+const app = express()
+app.use('/getBridgeLimits', getBridgeLimitsRoute)
+
+describe('Get Bridge Limits Route', () => {
+  it('should return min/max origin amounts for valid input', async () => {
+    const response = await request(app).get('/getBridgeLimits').query({
+      fromChain: 1,
+      fromToken: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
```

Comment:
import `USDC` from bridgeable vs. hardcoding address where we already have it in app